### ISSUE TITLE
[info] How to access encrypted 030 drives on another computer

### ISSUE THREAD

**Issue by chrisguida** [Start9Labs/embassy-os#962]
I want to be able to copy data directly to my hard drive rather than over the network. This will result in being able to transfer, for example, an archival bitcoind (~400GB) to my embassy in under an hour, rather than having to wait over 12 hours for it to transfer over then network.

This will especially help immensely with development.

How can I achieve this?

**Comment by ProofOfKeags** [comment]
Looks like this was duplicated in #963

**Comment by chrisguida** [comment]
Nope, not a duplicate. One is for encrypted drives, the other is for encrypted backups.

**Comment by MattDHill** [comment]
Important, but not pertinent to 0.3.0 release. Removing from 0.3.0 project board.

**Comment by dr-bonez** [comment]
this is fairly complicated, and likely is a documentation task rather than a dev task. Process by which it is mounted can be found in code at [appmgr/src/disk/main.rs](https://github.com/Start9Labs/embassy-os/blob/integration/0.3.0/appmgr/src/disk/main.rs)
